src/app/admin/time-table/time-table.component.ts

```diff
@@ -179,9 +196,11 @@ export class TimeTableComponent implements OnInit {
         this.table.renderRows();
         this.dataSource.paginator = this.paginator;
       }
-    }, (error: any) => {
-      if (error.error.response.includes('Wrong input')) {
+    }, (err: any) => {
```

Comment:
This method is called when observable is completed. Are you sure you need this callback here?